recipes-openxt/qemu-dm/qemu-dm.inc

```diff
@@ -22,6 +22,7 @@ SRC_URI = "http://wiki.qemu-project.org/download/qemu-${PV}.tar.bz2"
 
 # QEMU Patch Queue
 SRC_URI += "file://0001-generic-xenstore-extensions.patch \
+            file://0001-readonly-ide.patch;striplevel=1 \
```

Comment:
It has no dependencies, and can safely apply in parallel at the top.  Ordering is implied by the list so the numbers should simply be used to indicate the dependency tree, IMO.